test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -100,25 +127,35 @@ describe("UserWallet - Lightning Pay", () => {
       amount: amountInvoice,
     })
 
+    // WIP FIXME: Fails because it expects GetTransactions... to be
+    //            ordered by latest first in array
```

Comment:
A comment I added that can probably be removed now that it looks like the test works with the new changes.
```suggestion
```